### PR TITLE
add created_by for tasks

### PR DESCRIPTION
**Pull request by Kasvit**
**STATUS**
READY

**DESCRIPTION**

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [ ] code coverage >90%
- [ ] DB indexes added if needed
- [x] seed.rb file is updated if needed
- [ ] PR is reviewed manually again (to make sure you have 100% ready code)
- [ ] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs

### CODE REVIEW

**Comment by Kasvit** [comment]
image

File: app/controllers/account/tasks_controller.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@ def new
   def create
     @project = parent
     @task = @project.tasks.build(tasks_params)
+    @task.created_by_id = current_user.id
```
Comment (igorkasyanchuk) [inline app/controllers/account/tasks_controller.rb]: @task.created_by = current_user

File: app/views/account/tasks/show.html.slim [hunk 2]
```diff
@@ -9,11 +9,19 @@
       .level-right
         .level-item id="watch_task_#{@task.id}"
           = render('/account/tasks/watch_link', task: @task, project: @project)
+    - if @task.created_by_id.present?
```
Comment (igorkasyanchuk) [inline app/views/account/tasks/show.html.slim]: created_by,present?

File: app/views/account/tasks/show.html.slim [hunk 3]
```diff
@@ -9,11 +9,19 @@
...
+      h6 = "Created by #{ User.where(id: @task.created_by_id).first.full_name}"
```
Comment (igorkasyanchuk) [inline app/views/account/tasks/show.html.slim]: this code ` User.where(id: @task.created_by_id).first.full_name` sucks!!!

File: app/views/account/tasks/show.html.slim [hunk 4]
```diff
@@ -9,11 +9,19 @@
...
     h3 Description
     = @task.description
+
     - if @task.due_date.present?
       h3 Due date
       = l @task.due_date, format: :date_month_year_concise
+
+    - if @task.completed_at.present?
+      h3 Completed at
+      = "#{l @task.completed_at, format: :date_month_year_concise} by #{User.where(id: @task.completed_by_id).first.full_name}"
```
Comment (igorkasyanchuk) [inline app/views/account/tasks/show.html.slim]: and here too

**Review by igorkasyanchuk — APPROVED** [review]

**Pull request merged**